UAS.Client/src/components/SeasonMatchOverview.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="match">
+    <a href="/" target="_blank" rel="noopener noreferrer" class="match__detail-wrapper">
+      <div class="match__team match__team--left">
+        <p class="match__team-name">Illuminar Gaming</p>
+        <div class="match__team-logo-wrapper">
+          <img
+            src="https://gamepedia.cursecdn.com/lolesports_gamepedia_en/thumb/5/58/Illuminar_Gaminglogo_square.png/1200px-Illuminar_Gaminglogo_square.png"
```

Comment:
We will need images as local files in near future (resized and compressed images), for now it is o.k

Reply:
It is obvious. I just put them for mock-up purpose.